game.py

```diff
@@ -559,6 +602,13 @@ def handleMessage(self, player, message):
 
             show_trail: True/False
 
+            maze_hole: True/False
```

Comment:
I'm counting.  Four different names for holes enabled.  :grin:

Reply:
😄 Sorry about that. I defined the names as when I needed them and thus ended up with different names.
I have changed most of the variable names to `add_hole` for simplicity.